src/library/rekordbox/rekordboxfeature.cpp

```diff
@@ -243,6 +242,44 @@ void insertTrack(
     QString comment = getText(track->comment());
     QString tracknumber = QString::number(track->track_number());
     QString anlzPath = devicePath + getText(track->analyze_path());
+    mixxx::RgbColor::optional_t color = mixxx::RgbColor::nullopt();
+
+    switch (track->color_id()) {
+    case 1:
+        // Pink
+        color = mixxx::RgbColor::optional(QColor(248, 112, 248));
```

Comment:
Can you use 0xRRGGBB or the `qRgb` macro instead of QColor?

Reply:
You could define the predefined colors as `constexpr mixxx::RgbColor` constants in an anonymous namespace.

Reply:
Replacing comments with actual code is usually a desirable effect.